docker-prod/Dockerfile

```diff
@@ -24,6 +25,21 @@ RUN if [ "${TARGETARCH}" = "amd64" ]; then \
   fi; \
   apk add --force-overwrite /tmp/glibc.apk && \
   apk add /tmp/glibc-bin.apk && \
+  # Install awscli
+  if [ "${TARGETARCH}" = "amd64" ]; then \
+    curl -sL https://awscli.amazonaws.com/awscli-exe-linux-x86_64-${AWSCLI_VERSION}.zip -o awscliv2.zip; \
+  elif [ "${TARGETARCH}" = "arm64" ]; then \
+    curl -sL https://awscli.amazonaws.com/awscli-exe-linux-aarch64-${AWSCLI_VERSION}.zip -o awscliv2.zip; \
+  fi; \
+  unzip awscliv2.zip && \
+  aws/install && \
```

Comment:
Does this work? Afaik awscli v2 can't run on alpine musl 🤔

Reply:
It does, look above in the Dockerfile, it installs glibc. That was already there, so I though "meh, let's try it" and it indeed works :+1: